### DISCUSSION TITLE
Keypad Support

### DISCUSSION THREAD

**Discussion by jordanphstein** [homebridge-simplisafe3/homebridge-simplisafe3#207] [Q&A] [answered]
I see that the Keypad is not officially supported, but I've noticed that sometimes when I ARM and DISARM the system, the status is sent back to my homebridge correctly, triggering my automation. Is there anyway to improve the reliability?

**Comment by shamoon** [comment, marked as answer]
The readme is a little confusing, "supporting" the keypad has no relevance to arming/disarming the system, that is completely supported. Im guessing the issue youre having is related to your other post.

**Reply by jordanphstein** [reply]
I don't think it is actually related, but let me keep working on it. It could be the fact that my iPad, which is acting as a Home Kit base is very slow... Not entirely sure yet.
